CHANGELOG.md

```diff
@@ -1,5 +1,49 @@
 # Changelog
 
++ **1.1.2** - _02/06/2018_
```

Comment:
@shirish-pampoorickal, v 1.1.2 works for me if that's what you want here. 👍 

If we want to be semantic here, though, we "should" bump the minor version number when adding a feature.

Doesn't really matter here. Just thought I'd point it out. Perhaps when we have OpenAPI specification stuff, we'd want to bump v2?